heron/schedulers/src/java/com/twitter/heron/scheduler/marathon/MarathonScheduler.java

```diff
@@ -138,6 +147,27 @@ protected String getTopologyConf(PackingPlan packing) {
     return appConf.toString();
   }
 
+  // build the container object
+  protected ObjectNode getContainer(ObjectMapper mapper) {
+    ObjectNode containerNode = mapper.createObjectNode();
+    containerNode.put(MarathonConstants.CONTAINER_TYPE, "DOCKER");
+    containerNode.set("docker", getDockerContainer(mapper));
+
+    return containerNode;
+  }
+
+  protected ObjectNode getDockerContainer(ObjectMapper mapper) {
+    ObjectNode dockerNode = mapper.createObjectNode();
+
+    dockerNode.put(MarathonConstants.DOCKER_IMAGE, "ndustrialio/heron-executor:jre8");
```

Comment:
Yes, that makes sense. I think it would also be a good idea to actually include the Dockerfile as well. Could I put that file in the /docker directory and name it `Dockerfile.executor` maybe?